src/index.ts

```diff
@@ -112,6 +112,16 @@ function escape(str: string) {
   return str.replace(/[.+*?^${}()[\]|/\\]/g, "\\$&");
 }
 
+/**
+ * Format error so it's easier to debug.
+ */
+function errorMessage(message: string, originalPath: string | undefined) {
+  if (originalPath) {
+    return `${message}: ${originalPath}; visit ${DEBUG_URL} for more info`;
```

Comment:
Not tied to this format, happy to get other opinions. Another PR used `${originalPath}: ${message}` format instead.